<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetPlot2D</class>
 <widget class="QWidget" name="WidgetPlot2D">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1149</width>
    <height>783</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/******************** QGroupBox ***********************/
QGroupBox#plotUiGroup, #plotShowGroup, #plotOperGroup, #plotSetupGroup
{
    min-width: 150px;
    max-width: 150px;

    min-height: 100px;
    max-height: 100px;

    border: 1px solid darkGreen;
    border-radius: 8px;
    background: rgba(250, 240, 230, 0.7);  /*背景半透明 */
    margin: 5px 0px 0px 0px;
}

QGroupBox#plotGroup
{
    border: 1px solid darkGreen;
    border-radius: 3px;
    margin: 5px 0px 0px 0px;
}

QGroupBox::title
{
    subcontrol-origin: margin;
    subcontrol-position: top;
}

/****************** QPushButton *******************/
QPushButton#clearPBtn, #fullShowPBtn, #savePBtn, #openPBtn, #pausePBtn
{
    border: 1px solid black;
    background-color: lightGray;
    width: 25px;
    height: 20px;
    border-radius: 5px;                 /* 边框圆角半径 */
}

QPushButton#clearPBtn:hover, #fullShowPBtn:hover, #savePBtn:hover, #openPBtn:hover, #pausePBtn:hover
{
    border: 1px solid darkCyan;
    border-radius: 5px;                 /* 边框圆角半径 */
}

QPushButton#clearPBtn:pressed, #fullShowPBtn:pressed, #savePBtn:pressed, #openPBtn:pressed, #pausePBtn:pressed
{
    color: white;
    border: 1px solid darkCyan;
    background-color: darkCyan;             /* 背景 */
    border-radius: 5px;                 /* 边框圆角半径 */
}


/***********************QScrollBar****************************/



</string>
  </property>
  <widget class="QPushButton" name="savePBtn">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>660</y>
     <width>63</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pausePBtn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>660</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>暂停</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/image/pause.png</normaloff>:/image/pause.png</iconset>
   </property>
  </widget>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>0</y>
     <width>177</width>
     <height>351</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>177</width>
     <height>0</height>
    </size>
   </property>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>841</width>
     <height>641</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QCustomPlot" name="customPlot" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>700</width>
        <height>500</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QScrollBar" name="horizontalScrollBar">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>35</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>主题</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="themeCombo"/>
       </item>
       <item>
        <widget class="QCheckBox" name="fullShowCheck">
         <property name="text">
          <string>全程</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="autoRangeCheck">
         <property name="text">
          <string>自适应量程</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="subGridCheck">
         <property name="text">
          <string>显示子网格</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>时间轴：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="timeAxisSpin">
         <property name="suffix">
          <string> s</string>
         </property>
         <property name="prefix">
          <string/>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearPBtn">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>清除</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="fullShowPBtn">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>整图</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="statusLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>绘图状态</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>../../qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
